src/flavors/raingardens/config.yml

```diff
@@ -158,6 +159,36 @@ place_types:
           iconAnchor: [25, 60]
           shadowSize: [50, 60]
           shadowAnchor: [14, 60]
+  idea:
+    label: _(idea)
```

Comment:
Same here, this `idea` label is not the same as our `Idea` label in our duwamish flavor. Maybe we should capitalize all place types?